content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -49,6 +49,13 @@ RuntimeClass in the **node.k8s.io/v1beta1** API version will no longer be served
 * All existing persisted objects are accessible via the new API
 * No notable changes
 
+#### PodSecurityPolicy {#psp-v125}
+
+PodSecurityPolicy in the **policy/v1beta1** API version will no longer be served in v1.25, and the PodSecurityPolicy admission controller will be removed.
+
+PodSecurityPolicy replacements are still under discussion, but current use can be migrated to
```

Comment:
:wave: @tallclair .
I'm a bit surprised that this page includes information about future feature deprecations (about 1 yr away?).

nit: PodSecurityPolicy replacements are still under discussion; however, you can migrate current PodSecurityPolicy resources  to ...

Reply:
the ones already included in the page are current deprecations (if you `kubectl get` those resources on a 1.20 API server, you get a message that they _are_ deprecated, when they'll be removed, and what you should switch to using)

Reply:
> current deprecations

makes sense to me

We could merge this change at the same time as the deprecation announcement.